alot/account.py

```diff
@@ -20,6 +20,7 @@ class StoreMailError(Exception):
 
 
 class Account(object):
+
```

Comment:
What about the empty line below the docstring? (I assume you meant the other docstring when you said "below")